mailer.js

```diff
@@ -61,8 +67,8 @@ module.exports = function (log) {
     this.mailer.close()
   }
 
-  Mailer.prototype._supportLinkAttributes = function () {
-    return linkAttributes(this.supportUrl)
+  Mailer.prototype._supportLinkAttributes = function (template) {
+    return linkAttributes(this._generateUTMLink(this.supportUrl, {}, template, 'support'))
```

Comment:
`support` here is vague and won't allow us to determine which email produced the click. Can `support` be tied to the template somehow, e.g., `activate-support` when called from `verifyEmail`?